src/store/middleware/inputMiddleware.js

```diff
@@ -42,6 +42,14 @@ function updateDrive(prevComputedInput, computedInput, dispatch) {
       dispatch(requestDrive({ straight, steer }));
     }
   }
+  const prevActiveSuspension = prevComputedInput.drive.activeSuspension;
+  const activeSuspension = computedInput.drive.aciveSuspension;
+  if (activeSuspension !== prevActiveSuspension) {
+    dispatch(requestJointPower({
+      "jointName": "aciveSuspension",
+      power: computedInput.drive.activeSuspension
```

Comment:
We've already stored `computedInput.drive.activeSuspension` into `activeSuspension`, we might as well use that variable to make this shorter/cleaner.